client/beefy/src/worker.rs

```diff
@@ -238,9 +244,17 @@ pub(crate) struct BeefyWorker<B: Block, BE, C, P, R, N> {
 	/// Best block a BEEFY voting round has been concluded for.
 	best_beefy_block: Option<NumberFor<B>>,
 	/// Buffer holding votes for future processing.
-	pending_votes: BTreeMap<NumberFor<B>, Vec<VoteMessage<NumberFor<B>, AuthorityId, Signature>>>,
+	pending_votes: BoundedBTreeMap<
+		NumberFor<B>,
+		Vec<VoteMessage<NumberFor<B>, AuthorityId, Signature>>,
```

Comment:
```suggestion
		BufferedVec<VoteMessage<NumberFor<B>, AuthorityId, Signature>, ConstU32<MAX_BUFFERED_VOTES_PER_ROUND>>,
```

Reply:
For the `BufferedVec` I am guessing you mean `BoundedVec` right?